python/packages/ai/teams/ai/clients/llm_client.py

```diff
@@ -68,13 +75,29 @@ class LLMClientOptions:
     Optional. When set the model will log requests
     """
 
+    start_streaming_message: Optional[str] = ""
```

Comment:
if the default is "", should it be an optional type?

Reply:
yup! The keyword `Optional` makes it optional